### DISCUSSION TITLE
有没有已经在做的 对应的后端项目

### DISCUSSION THREAD

**Discussion by adoin** [zyronon/douyin#116] [Q&A]
有没有已经在做的 对应的后端项目

**Comment by thalesmark** [comment]
我在做了 是django写的

**Comment by thalesmark** [comment]
@adoin 有兴趣合作开发么？后端我写的差不多了

**Reply by huar96767** [reply]
怎么参与。

**Reply by adoin** [reply]
可我是前端 会一点 Java
django 不一样吧

**Reply by thalesmark** [reply]
> 可我是前端 会一点 Java django 不一样吧

后端完全我来负责 合作的人会前端就行 不需要会后端，有兴趣加我QQ 907087754

**Reply by thalesmark** [reply]
> 怎么参与。

加我QQ 907087754

**Reply by adoin** [reply]
> > 怎么参与。
> 
> 加我QQ 907087754

细聊

**Comment by elarbor** [comment]
后端做出来记得更新项目[readme](https://github.com/zyronon/douyin?tab=readme-ov-file#--douyin-vue)文件啊
